src/pages/post-feed/icons.tsx

```diff
@@ -1,9 +1,9 @@
 import Image from 'next/image';
-import facebook from '../../../public/feedPostIcons/facebook.svg';
-import Linkedinicon from '../../../public/feedPostIcons/LinkedInIcon.svg';
-import Twittericon from '../../../public/feedPostIcons/TwitterIcon.svg';
-import Email from '../../../public/feedPostIcons/Email.svg';
-import sandwichButton from '../../../public/feedPostIcons/SandwichButton.svg';
+import facebook from '../../../public/media/icons/facebook.svg';
+import Linkedinicon from '../../../public/media/icons/LinkedInIcon.svg';
+import Twittericon from '../../../public/media/icons/TwitterIcon.svg';
+import Email from '../../../public/media/icons/Email.svg';
+import sandwichbutton from '../../../public/media/icons/sandwichButton.svg';
```

Comment:
❌ 
you have changed the path, that is awesome

but you didn't move the files to the correct path, right?

Reply:
Nope, I did, and everything worked correctly. Should I stage all SVGs again? 🤷‍♀️😁

Reply:
if this works then something is wrong, maybe you are not using this file at all, it could be something else

I order to figure this out, you need to answer the question, what does this file do?
are you even using it anywhere ?

Reply:
I answered this below.